app/src/main/java/com/courseproject/tindar/presenters/viewprofiles/ViewProfilesPresenter.java

```diff
@@ -0,0 +1,4 @@
+package com.courseproject.tindar.presenters.viewprofiles;
+
+public class ViewProfilesPresenter {
```

Comment:
If you don't need this class, you can delete this file and the folder it belongs to. If you are planning to add something here later, pls add a TODO comment.